interni/verbali/verbale-interno_2020-03-24.tex

```diff
@@ -0,0 +1,93 @@
+\documentclass{article}
+
+\input{../../commons/config}
+
+\appendToGraphicspath{../../commons/img/}
+
+\title{Verbale interno --- 25/03/2020}
+
+\setResponsabile{Luca Ercole}
+\setRedattori{Riccardo Agatea}
+\setVerificatori{
+  Tobia Apolloni
+}
+\setUso{Interno}
+\setDescrizione{Verbale dell'incontro di GruppOne del 11/03/2020}
+\setModifiche{%
+\cellcolor{white!80!lightgray!100} & Luca Ercole & 2020--03--28 & approva documento \\%
+\cellcolor{white!80!lightgray!100} & Verificatore & 2020--03--26 & verifica verbale \\%
+\multirow{-3}{*}{-} & Riccardo Agatea & 2020--03--25 & stendi verbale%
+}
+
+\disabilitaVersione{}
+\disabilitaElencoFigure{}
+\disabilitaElencoTabelle{}
+
+\begin{document}
+
+\input{../../commons/template}
+
+\section{Informazioni logistiche}%
+\label{sec:informazioni_logistiche}
+
+\begin{description}
+  \item [Luogo] videochiamata su Google Hangouts
+  \item [Data] 25/03/2020
+  \item [Ora] 15:00 \symbol{8594} 17:00
+\end{description}
+
+\subsection{Membri del gruppo presenti}%
+\label{sub:membri_del_gruppo_presenti}
+
+\begin{enumerate}
+  \item Riccardo Agatea
+  % \item Tobia Apolloni
+  \item Riccardo Cestaro
+  \item Alberto Cocco
+  \item Luca Ercole
+  \item Alberto Gobbo
+  \item Alessandro Rizzo
+  \item Fabio Scettro
+\end{enumerate}
+% sub:membri_del_gruppo_presenti (end)
+
+\section{Ordine del giorno}%
+\label{sec:ordine_del_giorno}
+
+\begin{itemize}
+  \item Punto della situazione
+  \item Finalizzazione POC
+\end{itemize}
+
+\section{Punto della situazione}%
+\label{sec:punto_della_situazione}
+
+Ciascuno dei tre team in cui ci siamo suddivisi per parallelizzare il lavoro nelle tre componenti del prodotto ha aggiornato gli altri sui propri avanzamenti.
+Gli endpoint del server sono definiti, rimane da decidere quali implementare e in che modo (cioè se implementarli effettivamente, oppure se ricorrere a dei mock) per il POC\@.
```

Comment:
Aggiungere POC a glossario?

Reply:
Lo aggiungiamo come acronimo di Proof of Concept e nel glossario lo colleghiamo alla definizione del termine